Makefile

```diff
@@ -327,9 +322,16 @@ bundle: bundle-prepare ## Generate final bundle files.
 	if [ $$(echo $${VALIDATION_OUTPUT} | grep -i 'warning' | wc -c) -gt 0 ]; then echo "please correct warnings and errors first"; exit -1 ; fi \
 	'
 
+.PHONY: update-bundle
+update-bundle: IMG=$(IMAGE_TAG_BASE):$(OPERATOR_VERSION)
+update-bundle: bundle ## Prepare a clean bundle to be commited
+
 .PHONY: bundle-build
 bundle-build: ## Build the bundle image.
+	cp ./bundle/manifests/netobserv-operator.clusterserviceversion.yaml tmp-bundle
+	$(SED) -i -r 's~:created-at:~$(DATE)~' ./bundle/manifests/netobserv-operator.clusterserviceversion.yaml
 	$(OCI_BIN) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+	mv tmp-bundle ./bundle/manifests/netobserv-operator.clusterserviceversion.yaml
```

Comment:
If the Docker build fails (line 333), then it won't move the file back to its original state.  An improvement is to put a leading - so it ignores any error and hence, the next statement will run.  
 `-$(OCI_BIN) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .`

The slight drawback is that it will say the error is ignored.  It's also not a complete solution since aborting with ctrl-c will still be a problem.  The best practice is to modify the copied file instead, but I understand it may not be easily done in this situation.

Reply:
I did what you suggested first - I think that's ok, we can live with the little drawback

Reply:
/lgtm